sycl/include/CL/sycl/handler.hpp

```diff
@@ -1062,7 +1062,7 @@ class __SYCL_EXPORT handler {
   }
 
 #ifdef SYCL_LANGUAGE_VERSION
-#define __SYCL_KERNEL_ATTR__ __attribute__((sycl_kernel))
+#define __SYCL_KERNEL_ATTR__ [[clang::sycl_kernel]]
```

Comment:
Spelling change is required to avoid compilation error when using MSVC custom host compiler